custom.config.xml

```diff
@@ -0,0 +1,163 @@
+	<displayview>
+		{
+		"displayRTKInject": true,
+		"displayGPSOrder": true,
+		"displayHWIDs": true,
+		"displayADSB": true,
+		"displayName": 1,
+		"displaySimulation": true,
+		"displayTerminal": true,
+		"displayDonate": true,
+		"displayHelp": true,
+		"displayAnenometer": true,
+		"displayQuickTab": true,
+		"displayPreFlightTab": true,
+		"displayAdvActionsTab": true,
+		"displaySimpleActionsTab": false,
+		"displayGaugesTab": true,
+		"displayStatusTab": true,
+		"displayServoTab": true,
+		"displayScriptsTab": true,
+		"displayTelemetryTab": true,
+		"displayDataflashTab": true,
+		"displayMessagesTab": true,
+		"displayRallyPointsMenu": true,
+		"displayGeoFenceMenu": true,
+		"displaySplineCircleAutoWp": true,
+		"displayTextAutoWp": true,
+		"displayCircleSurveyAutoWp": true,
+		"displayPoiMenu": true,
+		"displayTrackerHomeMenu": true,
+		"displayCheckHeightBox": true,
+		"displayPluginAutoWp": true,
+		"displayInstallFirmware": true,
+		"displayWizard": true,
+		"displayFrameType": true,
+		"displayAccelCalibration": true,
+		"displayCompassConfiguration": true,
+		"displayRadioCalibration": true,
+		"displayEscCalibration": true,
+		"displayFlightModes": true,
+		"displayFailSafe": true,
+		"displaySikRadio": true,
+		"displayBattMonitor": true,
+		"displayCAN": true,
+		"displayCompassMotorCalib": true,
+		"displayRangeFinder": true,
+		"displayAirSpeed": true,
+		"displayPx4Flow": true,
+		"displayOpticalFlow": true,
+		"displayOsd": true,
+		"displayCameraGimbal": true,
+		"displayMotorTest": true,
+		"displayBluetooth": true,
+		"displayParachute": true,
+		"displayEsp": true,
+		"displayAntennaTracker": true,
+		"displaySerialPorts": true,
+		"displayBasicTuning": true,
+		"displayExtendedTuning": true,
+		"displayStandardParams": true,
+		"displayAdvancedParams": true,
+		"displayFullParamList": true,
+		"displayFullParamTree": true,
+		"displayParamCommitButton": false,
+		"displayBaudCMB": true,
+		"displaySerialPortCMB": true,
+		"standardFlightModesOnly": false,
+		"autoHideMenuForce": false,
+		"displayInitialParams": true,
+		"isAdvancedMode": true,
+		"displayServoOutput": true,
+		"displayJoystick": true,
+		"displayOSD": true,
+		"displayUserParam": true,
+		"displayPlannerSettings": true,
+		"displayFFTSetup": true,
+		"displayPreFlightTabEdit": true,
+		"displayPlannerLayout": true,
+		"lockQuickView": false
+		}
+	</displayview>
+	<enableadsb>False</enableadsb>
+	<FPaltmode>3</FPaltmode>
+	<fpcoordmouse>GEO</fpcoordmouse>
+	<fpminaltwarning>0</fpminaltwarning>
+	<groundColorToolStripMenuItem>False</groundColorToolStripMenuItem>
+	<guided_alt>100</guided_alt>
+	<HUD_showicons>True</HUD_showicons>
+	<hud1_useritem_DistToHome>DistToHome: </hud1_useritem_DistToHome>
+	<hud1_useritem_sonarrange>Lidar: </hud1_useritem_sonarrange>

```

Comment:
kind of hard to review but I did find a difference here. I have 10 preceding spaces to have the message in the right location